libsplinter/src/network/peer_manager/connector.rs

```diff
@@ -97,6 +97,37 @@ impl PeerManagerConnector {
             .map_err(|err| PeerRefAddError::ReceiveError(format!("{:?}", err)))?
     }
 
+    /// Request that a peer is added to the PeerManager. If a peer already exists, the peer's ref
+    /// count will be incremented. This function should be used when the peer id is unknown.
+    ///
+    /// # Arguments
+    ///
+    /// * `endpoint` -  The  endpoints associated with the peer.
+    ///
+    /// Returns a PeerRef, that when dropped, will automatically send a removal request to the
+    /// PeerManager.
+    pub fn add_unidentified_peer(&self, endpoint: String) -> Result<PeerRef, PeerRefAddError> {
```

Comment:
The commit message doesn't reflect this change.

Reply:
Whoops, fixed (old commit message)